templates/github/ci/.github/workflows/release.yml

```diff
@@ -120,10 +120,10 @@ jobs:
       - name: Lint plugin
         run: |
           git clone https://github.com/grafana/plugin-validator
-          pushd ./plugin-validator/cmd/plugincheck
+          pushd ./plugin-validator/cmd/plugincheck2
```

Comment:
I'm confused by this path. Shouldn't it be `./plugin-validator/pkg/cmd/plugincheck2` as per the repo https://github.com/grafana/plugin-validator/tree/master/pkg/cmd/plugincheck2 ??

Reply:
yes it should be that. I wonder if this `pushd` is failing and the script goes on? Though `pushd` exits with code 1 when the path is not found.

Reply:
It should be the path as outlined by @jackw

while we at it git clone https vs ssh?

Reply:
Ok, I've updated the path. What's the benefit of using ssh over https here @tolzhabayev ? 🤔

Reply:
No idea, just used to `git clone` using `git@` <- ignore my comment it does not seem to add any value